app/javascript/react/components/ReviewTile.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import Enzyme, { mount } from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+
+Enzyme.configure({ adapter: new Adapter() })
```

Comment:
I wouldn't expect you to need anything above line 6 -- can you try removing it and seeing if your tests still run? 
* If they do still run, please delete these lines, push, and make sure Codeship passes. You then have my permission to merge this branch into master ("press the big green button").
* If they don't run, you can go ahead and merge this branch into master 🙂